docs/examples/Tile.jsx

```diff
@@ -1,6 +1,6 @@
 var image = {
   src: 'http://brand.holidayextras.com/img/product-main.jpg',
-  alt: 'Two course meal'
+  alt: 'Tender sirloin steak with blue cheese dressing'
```

Comment:
Should we change `<UIToolkit.Tile image={image} title` on line 7, to also equal this?